templates/CRM/Twingle/Page/Profiles.tpl

```diff
@@ -34,10 +34,16 @@
       <tbody>
       {foreach from=$profiles item=profile}
         {assign var="profile_name" value=$profile.name}
-        <tr>
+        <tr style="border-bottom: 1px solid #cfcec3;">
           <td>{$profile.name}</td>
           <td>
-            <div><strong>{ts domain="de.systopia.twingle"}Selector{/ts}:</strong> {$profile.selector}</div>
+              {if not $profile.is_default}
```

Comment:
Why does this not apply for the default profile?

Reply:
The default profile takes care of any submission containing a `project_id` (selector) that doesn't match one of the other profiles.

Reply:
But it could still have selectors configured, right? So I'd suggest displaying them would still make sense.

Reply:
I still don't understand the point of assigning selectors to the default profile.

Reply:
Did you see my PR description for #74?

Reply:
People can do it and do it - if only for having it noted somewhere or when having to copy and paste it to another profile later - so why not display their config where it's supposed to be?

Reply:
Ok, this indeed makes sense with #74, sorry. I'm not sure I then really like the idea, though ... I think I'll have someone else review it ...

Reply:
I find this confusing. If people can assign project_ids to the default profile, they might think that they have to do that in order for the default profile to work. This impression might be reinforced by the fact that so far they are forced to enter a value in the field to be able to save the profile.

Reply:
I asked @bjendres for another pair of eyes on that. I think you're right, but I'd like to re-check.